### PR TITLE
Feature addanchors

### PR DESCRIPTION
**Pull request by mb2055**
Created the "scripts" folder. 
Currently contains the "addanchors.py" script which creates a modified system with dummy atoms designed to be used in cartesian restraints. It also creates a "restraint.cfg" file which can be pasted directly in to a SOMD (latest version) config file in order to create the restrained atom pairs.
Also included is a jupyter notebook that shows an example of selecting heavy atoms within a given range of a ligand using sire.

### CODE REVIEW

**Review by lohedges — APPROVED** [review]
This looks good to me. Shall we wait for feedback from Cresset before merging, i.e. in case modifications are needed? I'm happy either way.

**Comment by mb2055** [comment]
I did test the functionality before telling Tom, but there may be unforeseen issues.

**Comment by lohedges** [comment]
Yes, it's not to see if it works as is (I tested too), rather whether it does what they want ;-)

**Pull request merged**
